### PR TITLE
Add various design improvements

### LINKED ISSUE #21: Use "object.og_image" method for OG image
Currently auto-rendered product pages use product image directly for OG image. As the product image is returned with media.voog.com URL, it is blocked on some social media platforms. Recently added `object.og_image` method fixes this issue.

### LINKED ISSUE #22: Add titles to content area tags
Add titles and tooltips to content area tags to help differentiate content areas.

### PR DESCRIPTION
**Pull request by PeeterPaal**
- Use `object.og_image` method for OG image
- Add titles & tooltips to various content areas

Closes #21 
Closes #22

### CODE REVIEW

File: components/site-footer.tpl [hunk 1]
```diff
@@ -51,7 +51,11 @@
           {% include "footer-menu" %}
         </div>
         <div class="footer-contact-wrap">
-          <div class="footer-contact content-area">{% xcontent name="footer" %}</div>
+          <div class="footer-contact content-area">
+            {%- assign footer_content_title = "footer" | lce -%}
+            {%- assign footer_content_title_tooltip = "content_tooltip_all_pages_same_language" | lce -%}
+            {% xcontent name="footer" title_tooltip=footer_content_title_tooltip %}
```
Comment (tanelj) [inline components/site-footer.tpl]: `footer_content_title` is missing.
```suggestion
            {% xcontent name="footer" title=footer_content_title title_tooltip=footer_content_title_tooltip %}
```

**Review by tanelj — CHANGES_REQUESTED** [review]
Nice, but one title is missing.

**Review by pxska — APPROVED** [review]
Looks good!

**Review by tanelj — APPROVED** [review]
Nice!

**Pull request merged**